packages/vitest-file-snapshots/src/validation-file-matcher/matcher.ts

```diff
@@ -1,50 +1,40 @@
-import * as fs from "node:fs";
-import * as path from "node:path";
 import type { ExpectationResult, MatcherState } from "@vitest/expect";
 import "vitest";
```

Comment:
Not related to this branch, but is the global import `import "vitest";` needed?

Reply:
Good finding, the import is not needed. It's only needed in the final declaration such that the Vitest declarations are defined before the module augmentation. This was likely a leftover. I removed the obsolete import in 97e60dfc3bacd41338741717a5d9bec21c5b935b.